.lintstagedrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "**/*.{js,mjs,ts,tsx,md,mdx}": [
-    "turbo run lint:lint-staged --",
+    "node --run lint",
```

Comment:
These are two separate actions, are they not? One will lint all files, one only the staged?

Reply:
As far as I can say, it _is_ running lint against the files.

For me it was something really weird due to:

- lint:staged was a pseudo turbo command that invokes the `lint` command in a few packages
- so there was no actual lint command, and if the pseudo command is not updated we might forget to add it to new packages.
- lintstaged will automatically append the files that need to be linted (afaik)

Reply:
But you can checkout the branch and verify it indeeds only runs lint-staged against the changed files.